### PR TITLE
Remove babel 6 test from AppVeyor

### PR DESCRIPTION
**Pull request by shad-k**
Removes babel 6 test from AppVeyor. Will only run the test against babel 7.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -24,21 +24,6 @@ test_script:
   - node --version && npm --version
   - echo -- BABEL 7 --
```
Comment (Andarist) [inline appveyor.yml]: this echo is redundant now, as we are testing only against babel 7, you should remove it

**Comment by Andarist** [comment]
Thanks for this! I've left only a small comment, other than that it looks good!

**Comment by shad-k** [comment]
@Andarist removed the echo statement as well.

**Comment by Andarist** [comment]
Thanks!

**Pull request merged**